client/extensions/woocommerce/state/wc-api/reducer.js

```diff
@@ -2,7 +2,6 @@
  * Internal dependencies
  */
 import settingsGeneral from './settings/general/reducer';
-import error from './error-reducer';
```

Comment:
This line doesn't exist in master as it stands right now. Not sure why this is here.